system/config/system.yaml

```diff
@@ -15,6 +15,7 @@ http_x_forwarded:                                # Configuration options for the
   host: false
   port: true
   ip: true
+avatar_use_gravatar: false                       # If enabled and no custom user avatar is set, then gravatar is used as external avatar source. Possibly relevant to your privacy policy.
```

Comment:
To me, this is an issue. We cannot change the existing behavior in a patch release without consent from the site owner. If we do that, it's considered to be a bug.

For the new sites this change is good, but if this will be the default, we need to update configuration in the existing sites to set the variable to true.